### ISSUE TITLE
Can't build iroha on Ubunto 18.04

### ISSUE THREAD

**Issue by building39** [hyperledger/iroha#2002]

**Comment by building39** [comment]
The build instructions aren't working for me. Here's what I did:
```
~/git$ git clone -b develop https://github.com/hyperledger/iroha
Cloning into 'iroha'...
remote: Enumerating objects: 367, done.
remote: Counting objects: 100% (367/367), done.
remote: Compressing objects: 100% (304/304), done.
remote: Total 74990 (delta 145), reused 106 (delta 62), pack-reused 74623
Receiving objects: 100% (74990/74990), 51.04 MiB | 131.00 KiB/s, done.
Resolving deltas: 100% (56492/56492), done.
~/git$ cd iroha
~/git$ cmake --version
cmake version 3.13.2

CMake suite maintained and supported by Kitware (kitware.com/cmake).
~/git/iroha$ mkdir build; cd build; cmake ..; make -j$(nproc)
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is GNU 7.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- -DCMAKE_BUILD_TYPE=Debug
-- -DTESTING=ON
-- -DBENCHMARKING=OFF
-- -DFUZZING=OFF
-- -DCOVERAGE=OFF
-- -DPACKAGE_ZIP=OFF
-- -DPACKAGE_TGZ=OFF
-- -DPACKAGE_RPM=OFF
-- -DPACKAGE_DEB=OFF
-- -DENABLE_LIBS_PACKAGING=ON
-- -DSWIG_PYTHON=OFF
-- -DSWIG_JAVA=OFF
-- -DSUPPORT_PYTHON2=OFF
-- -DSWIG_CSHARP=OFF
-- -DSWIG_NODE=OFF
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found gtest: /usr/include  
-- Found spdlog: /usr/include  
-- Found protobuf: /usr/include  
-- Found grpc: /usr/local/lib/libgrpc.so  
-- Found rapidjson: /usr/include  
-- Found pq: /usr/include/postgresql  
-- Found soci: /usr/include/soci  
-- Found gflags: /usr/include  
-- Found rxcpp: /usr/local/include  
-- Found tbb: /usr/include  
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   filesystem
--   system
--   thread
--   chrono
--   date_time
--   atomic
-- Found ed25519: /usr/local/include  
-- Configuring done
-- Generating done
-- Build files have been written to: /home/mmartin/git/iroha/build
[  1%] Generating ../../schema/endpoint.pb.h, ../../schema/endpoint.pb.cc
endpoint.proto: No such file or directory
[  1%] Generating yac.grpc.pb.h, yac.grpc.pb.cc, yac_mock.grpc.pb.h
shared_model/schema/CMakeFiles/schema.dir/build.make:118: recipe for target 'schema/endpoint.pb.h' failed
make[2]: *** [schema/endpoint.pb.h] Error 1
CMakeFiles/Makefile2:3763: recipe for target 'shared_model/schema/CMakeFiles/schema.dir/all' failed
make[1]: *** [shared_model/schema/CMakeFiles/schema.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  1%] Generating yac.pb.h, yac.pb.cc
yac.proto: No such file or directory
schema/CMakeFiles/yac_grpc.dir/build.make:70: recipe for target 'schema/yac.grpc.pb.h' failed
make[2]: *** [schema/yac.grpc.pb.h] Error 1
make[2]: *** Waiting for unfinished jobs....
yac.proto: No such file or directory
schema/CMakeFiles/yac_grpc.dir/build.make:62: recipe for target 'schema/yac.pb.h' failed
make[2]: *** [schema/yac.pb.h] Error 1
CMakeFiles/Makefile2:98: recipe for target 'schema/CMakeFiles/yac_grpc.dir/all' failed
make[1]: *** [schema/CMakeFiles/yac_grpc.dir/all] Error 2
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
~/git/iroha/build$
```

**Comment by l4l** [comment]
Sorry, our docs are quite outdated. Current active development branch is `dev`, and for releases we use `master`. Issue should be fixed in #2009 

I hope that will help, feel free to add something if issue still there
